### ISSUE TITLE
Problem upgrading to Gantry 5.3.9

### ISSUE THREAD

**Issue by collector00** [gantry/gantry5#1739]
I hope this is the correct forum to post this. I recently upgrade to Gantry 5.3.9 and ever since I can no longer enter the Layouts of a specific Style. For example Template > Callisto-Default > Layout. Also, if I try to clear cache from within the template the icon just spins. Callisto is the template I'm using. I actually have it loaded temporarily as I investigate what I will need to do to upgrade my site from Gantry 4. The layout was working perfectly before I upgrade. I have checked just about everything I can think of and all signs point to an issue with the Gantry upgrade.

I am running Joomla 3.6.4, Gantry 5.3.6 and Callisto 1.3.6.

**Comment by blizam** [comment]
Did you flush out all your Joomla/Gantry system caches after the upgrade? Do you have Particle Defaults assigned on these Layouts? Particle Defaults should only be assigned on the Base Outline now... they used to be available on any Outline which never really made sense... migrate any Particle Default settings you may have on those Outlines elsewhere... either to the Layout directly or to the Base Outline's Particle Defaults

**Comment by mahagr** [comment]
Also what's the issue you get -- did you take a look into the ajax reply?

**Comment by collector00** [comment]
Hi, yes I cleared the Joomla cache but the clear cache within the template does not work - it is also spinning. The site has not been changed; I have made no changes so it is Callisto template with the demo material installed.  For haha's this morning  I loaded a Joomla and Callisto in a new folder and it works perfectly fine. Of course, I have not upgraded it. 

The issue is I cannot access Layouts. On occasion I may be able to get into layouts but I cannot save anything. When I click Save the icon just spins.
